examples/network-xr.yaml

```diff
@@ -1,11 +1,10 @@
 apiVersion: azure.platform.upbound.io/v1alpha1
 kind: XNetwork
 metadata:
-  annotations:
-    render.crossplane.io/composition-path: apis/pat/composition.yaml
-    render.crossplane.io/function-path: examples/functions.yaml
   name: ref-azure-network
 spec:
   parameters:
     id: ref-azure-network-from-xr
     region: westus
+    deletionPolicy: Delete
```

Comment:
You don't need this when using up composition render when you add the --xrd we will extract the defaults for the fields

Reply:
Should `--xrd` be the default?  :)

Reply:
Readme and sample XR updated